src/Microsoft.AspNetCore.Mvc.Abstractions/Filters/IResultFilter.cs

```diff
@@ -4,8 +4,24 @@
 namespace Microsoft.AspNetCore.Mvc.Filters
 {
     /// <summary>
-    /// A filter that surrounds execution of the action result.
+    /// A filter that surrounds execution of action results successfully returned from an action.
```

Comment:
Probably add similar comments for ResultFilterAttribute too?

Reply:
I've always thought those were around to support legacy ports. That said, it might be useful to support if you're trying to convert an existing ResultFilterAttribute derived type. I'd prefer if we wait for feedback, but maybe @rynowak feels strongly about it.

Reply:
Since ResultFilterAttribute supports ordering, a user might be tempted to just derive from it for that sake rather than going and creating their own new attribute or adding it through application model...so for the same reason, should we have an corresponding attribute for IAlwaysRunResultFilter too?

Reply:
I'd prefer not. The current attributes date back to old MVC, and do more than they advertise. See `ActionFilterAttribute`.  

It's a good idea, but I'd prefer not to add more of these when the ones we have are legacy with a capital L. Cleaning them up would be an improvement, but would require breaking changes.